source/developer/personal-access-tokens.rst

```diff
@@ -74,16 +74,16 @@ Frequently Asked Questions (FAQ)
 How do personal access tokens differ from regular session tokens?
 ..................................................................
 
-Personal access tokens do not expire. As a result, you can more easily integrate with Mattermost, bypassing the `session length limits set in the System Console <https://docs.mattermost.com/administration/config-settings.html#sessions>`_.
+- Personal access tokens do not expire. As a result, you can more easily integrate with Mattermost, bypassing the `session length limits set in the System Console <https://docs.mattermost.com/administration/config-settings.html#sessions>`_.
+- Personal access tokens can be used to authenticate against the API more easily, including with AD/LDAP and SAML accounts.
+- You can optionally assign additional roles for the account creating personal access tokens. This lets the account post to any channel in Mattermost, including direct messages.
 
-Moreover, you can optionally assign additional roles for the account creating personal access tokens, letting the account post to any channel in Mattermost, including direct message.
+Besides the above differences, personal access tokens are exactly the same as regular session tokens. They are cryptic random IDs and are not different from user's regular session token created after logging in to Mattermost.
 
-Should I be worried the personal access tokens last forever?
-.............................................................
+Can I have expiring personal access tokens?
+............................................
 
-No. Personal access tokens are cryptic random IDs and are not different from user's regular session token created after logging in to Mattermost.
-
-If you do continue to have worries, you can set up your integration to self-rotate the personal access tokens via `available API calls <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D~1tokens%2Fpost>`_, or simply use the regular user session tokens. The API calls support the new **post:all** and **post:channels** roles as well.
+Yes. You can set up your integration to self-rotate the personal access tokens via `available API calls <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D~1tokens%2Fpost>`_.
```

Comment:
I think we should be a little more clear that this is being done manually by their integration, not by us. Saying "Yes" to this is a bit confusing since it's really "No, but you can automate your integration to cycle its token through the REST API"